runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapPoint.java

```diff
@@ -39,6 +39,10 @@ public class WorldMapPoint
 
 	private WorldPoint worldPoint;
 
+	private WorldPoint worldPointToMapWorldPoint;
```

Comment:
Can annotate as nullable
The purpose of this is to "cache" the value right? so it starts as null & is fetched on demand, but next time it's fetched it'll already have the value